server/migration/1624566131004-AddObservation.ts

```diff
@@ -0,0 +1,18 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddObservation1624566131004 implements MigrationInterface {
+  name = 'AddObservation1624566131004';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`CREATE TABLE "observation" ("id" SERIAL NOT NULL, "document_id" character varying(100) NOT NULL, "edipi" character varying(10) NOT NULL, "timestamp" TIMESTAMP NOT NULL, "unit_id" character varying(50) NOT NULL, "unit" character varying(100) NOT NULL, "type_id" character varying, "type_org" integer, CONSTRAINT "UQ_c3b96ba3699ceaa1c9c95bdab10" UNIQUE ("id", "document_id"), CONSTRAINT "PK_77a736edc631a400b788ce302cb" PRIMARY KEY ("id"))`);
```

Comment:
Is it possible to annotate the model so that the default `type_org` becomes `org_id` or `type_org_id`?

Reply:
This is normally done by setting the `name` property on the `@Column` decorator's options. But I don't think `type_org` should even be showing up in the table like this. It probably has something to do with the erroneous "inverse side" setting in the `type` column.

Reply:
These columns where generated by TypeORM, see report_schema:

`CREATE TABLE IF NOT EXISTS public.report_schema
(
    id character varying COLLATE pg_catalog."default" NOT NULL,
    name character varying(256) COLLATE pg_catalog."default" NOT NULL,
    columns json NOT NULL DEFAULT '[]'::json,
    org_id integer NOT NULL,
    CONSTRAINT "PK_c19c025244ceedaff94736f021f" PRIMARY KEY (id, org_id),
    CONSTRAINT "FK_d1ffa8699456464f8d833838cba" FOREIGN KEY (org_id)
        REFERENCES public.org (id) MATCH SIMPLE
        ON UPDATE NO ACTION
        ON DELETE CASCADE
)
`

Reply:
It looks like `primary: true` is set on that column in `report-schema.model.ts`, so that's expected for the table to be generated like that.

The `id` on `ReportSchema` isn't automatically generated, so I suspect there's the possibility of id collisions, which is why `org_id` is part of the primary key. I can't remember the specific reason since I didn't write that `ReportSchema` model, but I recall JD saying this was necessary for some reason.

Reply:
Go it. As there is not action I am resolving this thread.